frontend/App.tsx

```diff
@@ -1,78 +1,134 @@
 import axios from 'axios';
-import { StatusBar } from 'expo-status-bar';
-import * as React from 'react';
-import { useEffect, useState } from 'react';
-import { StyleSheet, Text, View, Button } from 'react-native';
-// import { Button } from "react-bootstrap";
+import React, { Component } from "react";
+import { Pressable, StyleSheet, Text, View, Button } from 'react-native';
+
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+// import MaterialIcons from "@expo/vector-icons/MaterialIcons";
+
 import { SearchContacts } from './components/SearchContacts';
 import { ChatPanel } from './components/ChatPanel';
+import { Regstyles, Registration } from './components/Registration';
+import { readData, removeData } from './services/AsyncStorage';
+// import { PhoneinputTest } from './components/PhoneinputTest'; 
 
- 
-export default function App() {
-  // const url = "https://gner4se1je.execute-api.ap-south-1.amazonaws.com/Prod/func1"
-  const func1_url = "http://127.0.0.1:3000/func1";
-  const func2_url = "http://127.0.0.1:3000/func2";
-  
-  let [func1_resp, setFunc1Resp] = useState({});
-  let [func2_resp, setFunc2Resp] = useState({});
+const func1_url = "http://127.0.0.1:3000/func1";
```

Comment:
Ideally, if these are being reused, we should move these constants to Cosntants.tsx

Reply:
ok